### DISCUSSION TITLE
Website is down

### DISCUSSION THREAD

**Discussion by marinsiebert** [idaholab/moose#25149] [Q&A General]
Hi everyone, 
the mooseframework is reachable again, however I am not able to update my conda distribution. I even tried removing everything and starting from scratch but I keep getting the following error: 

conda install moose-tools
Collecting package metadata (current_repodata.json): failed

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.software.inl.gov/public/linux-64/current_repodata.json>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
'https//conda.software.inl.gov/public/linux-64'

It seems to me there are some server issues on the INL side, or am I missing something? 

Edit: it seems this is a similar problem as in #25148

**Comment by marinsiebert** [comment]
The Framework is now also down, again.

**Reply by milljm** [reply]
Thanks for reporting!

We've definitely been experiencing network issues here at the INL. As I walked in this morning, both resources appear to be functional once more.

**Reply by marinsiebert** [reply]
I have been having issues since yesterday, still everything dead on my side. Hopefully it will be better tomorrow.

**Reply by milljm** [reply]
I am using Verizon wireless, and can access both sites. Publicly, it appears available. What network service are you using?

**Comment by marinsiebert** [comment]
I have been trying from my office, home and mobile network, so i doubt it's a network issue (I should mention I am from Germany).

**Reply by milljm** [reply]
Oh that is interesting... when using this service: https://www.locabrowser.com/, and selecting Germany, I get a "No response" operation.

**Reply by milljm** [reply]
Canada works, Japan does not... I can't tell if this is somehow restricted from my machine or not. Still, I will report it to our network team...

**Reply by milljm** [reply]
@marinsiebert Can you post a screen shot of what occurs when you attempt to browse to https://mooseframework.inl.gov ?

I am being asked to provide some data while we attempt to resolve this issue. Thanks!!

**Reply by lindsayad** [reply]
To elaborate a little more, INL's "Information Management" division is asking us, the MOOSE team, to "prove it". Sorry for the hassle, but hopefully with your help we can help get through to them that they're making things a real pain for our broader global community

**Reply by marinsiebert** [reply]
![Screenshot_2023-08-09-22-21-01-836_com.android.chrome.jpg](https://github.com/idaholab/moose/assets/113979717/1f4c0532-78c7-4046-9083-826f97c9f8a0)


Right now it works fine again on my phone using wifi, mobile data stays silent however (screenshot attached).
Will try again in the morning on my laptop to reach the site and create the conda environment.

**Reply by marinsiebert** [reply]
As I posted this I was able to connect to the website, the rest I will try tomorrow

**Reply by lindsayad** [reply]
It's better for us to have the screenshot showing it's not working 😄 Thank you so much!

**Comment by marinsiebert** [comment]
Everything seems to working fine now as I am able to reach the website and update both conda and the framework without issues on my WSL (although I did get the same error during tests as was mentioned in #23042 ; exporting DISPLAY=:0.0 like described worked as a fixed for me)
